web/src/features/fireBehaviourCalculator/components/GrassCureCell.tsx

```diff
@@ -59,18 +72,8 @@ const GrassCureProps = (props: GrassCureCellProps) => {
       variant="outlined"
       inputProps={{ min: 0, maxLength: 4, size: 4, max: 100 }}
```

Comment:
btw the "maxLength" and "size" attributes only apply to textual fields, not numeric. I know it's not relevant to this PR; just FYI